README.md

```diff
@@ -196,23 +196,12 @@ Please note that you -must- provide a Connection String (`connection-string`) an
 
 ```hocon
 akka.persistence {
-  publish-plugin-commands = on
-
   journal {
-    plugin = "akka.persistence.journal.sql"
```

Comment:
Isn't this setting definitely needed in order to get Akka.Persistence to load the plugin correctly?

Reply:
No, this is a dump of the default hocon configuration. That line should NOT be in the default configuration file.
It is in the `Setup` section on line 23-38.